### PR TITLE
Processing promoter and control sequences

### PR DESCRIPTION
**Pull request by aaronwtr**
Processing 20k promoter and control sequences. Regions are stored in sequences folder as .bed. DNase values are stored in .pkl file in the same folder. We are using K562 DNase data.

### CODE REVIEW

File: eft/preprocess.py [hunk 1]
```diff
@@ -52,33 +75,41 @@ def main(args) -> None:
...
+                row_data = {
+                    'chrom': chrom,
+                    'start': start,
+                    'end': end,
+                    'values': values
+                }
+                control.append(row_data)
 
+    promoters = pd.DataFrame(promoters)
+    control = pd.DataFrame(control)
```
Comment (wconnell) [inline eft/preprocess.py:107]: I think we should label the sequence type and then just concat and shuffle these into a single file. We will be splitting this file into `train` and `val` after loading anyways:
```
promoter['seq_type'] = 'promoter'
control['seq_type'] = 'random'
sequences = pd.concat((promoter, control)).sample(frac=1)
```

File: eft/preprocess.py [hunk 2]
```diff
@@ -52,33 +75,41 @@ def main(args) -> None:
...
 
+    promoters.to_pickle(f"{outdir}/promoter_dnase_vals.pkl")
+    promoters[['chrom', 'start', 'end']].to_csv(f"{outdir}/promoter_dnase.bed", sep="\t", header=False, index=False)
+    control.to_pickle(f"{outdir}/control_dnase_vals.pkl")
```
Comment (wconnell) [inline eft/preprocess.py]: I don't think we need the pickled files any longer, lets remove

File: eft/models.py
Comment (wconnell) [inline eft/models.py:1]: lightning calls `forward` implicitly with `self()`, so we shouldn't need to change this file

File: eft/preprocess.py [hunk 3]
```diff
@@ -52,33 +75,41 @@ def main(args) -> None:
...
+                    'end': end,
+                    'values': values
+                }
+                control.append(row_data)
 
+    promoters = pd.DataFrame(promoters)
+    control = pd.DataFrame(control)
 
+    promoters.to_pickle(f"{outdir}/promoter_dnase_vals.pkl")
+    promoters[['chrom', 'start', 'end']].to_csv(f"{outdir}/promoter_dnase.bed", sep="\t", header=False, index=False)
```
Comment (wconnell) [inline eft/preprocess.py]: this should include `values` column

**Review by wconnell — CHANGES_REQUESTED** [review]
hey couple small requests per my preference; lmk if I'm overlooking something, thanks!

**Comment by aaronwtr** [comment]
@wconnell thank you, I agree on all suggestions! They have been refactored

**Review by wconnell — APPROVED** [review]

**Pull request merged**